packages/cursorless-engine/src/languages/TreeSitterQuery/QueryCapture.ts

```diff
@@ -38,6 +38,9 @@ export interface QueryCapture {
 
   /** The insertion delimiter to use if any */
   readonly insertionDelimiter: string | undefined;
+
+  /** The insertion prefix to use if any */
+  readonly insertionPrefix: string | undefined;
```

Comment:
Separate the insertion delimiter that is used to separate each item in a collection from the insertion prefix which only goes before the text content and that is used on every item even if the collection only has one. Hence this is not actually a delimiter.